src/main/java/seedu/address/model/role/Participant.java

```diff
@@ -0,0 +1,44 @@
+package seedu.address.model.role;
+
+import java.util.Set;
+
+import seedu.address.model.person.Address;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.privilege.Privilege;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Participant for FOP.
+ * <p>
```

Comment:
Is the p tag necessary?

Reply:
hmm, for both it wasn't there initially but somehow was added probably by the system. but thanks for the notice! xoxo